carbon/app.go

```diff
@@ -271,6 +272,7 @@ func (app *App) Start() (err error) {
 			receiver.WriteChan(app.writeChan),
 			receiver.DropFuture(uint32(conf.Udp.DropFuture.Value().Seconds())),
 			receiver.DropPast(uint32(conf.Udp.DropPast.Value().Seconds())),
+			receiver.DropLongerThan(conf.Tcp.DropLongerThan),
```

Comment:
@franek1709 Am I right and it's the bad copy/paste here and further? Should be Udp and so on?